### ISSUE TITLE
Add a enlarge leaflet option

### ISSUE THREAD

**Issue by trungng2006** [valentine195/obsidian-leaflet-plugin#43]
Usecase: Sometime I want to enlarge the leaflet to take full width (or full height) of my screen. (i usually would like to use this feature with the [maximize active pane plugin](https://github.com/deathau/maximise-active-pane-obsidian)).
Hence, would it be possible to to add this option? 
EX: ![image](https://user-images.githubusercontent.com/71961905/117578443-f573aa00-b120-11eb-8a2c-c0444b891c27.png)

**Comment by valentine195** [comment]
You would want the map to go full-screen, right? Not "maximize within the current view". Just wanted to make sure I understand the request correctly.

**Comment by trungng2006** [comment]
Yes, i would like the map to go full screen! (sorry for not being clear)

**Comment by trungng2006** [comment]
Thank you!

**Comment by valentine195** [comment]
I don't think this would be easy to implement due to the way the Modal API is setup through Obsidian. I'll tinker with it when I get some time, though, and see if I can force the containing element to be the map instead of the document body.

**Comment by trungng2006** [comment]
Thank you so much!
